schema/meter_provider.json

```
@@ -124,6 +130,102 @@
                     "type": "integer"
                 }
             }
+        },
+        "ViewRegistration": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "selector": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "instrument_name": {
+                            "type": "string"
+                        },
+                        "instrument_type": {
+                            "type": "string"
+                        },
+                        "meter_name": {
+                            "type": "string"
+                        },
+                        "meter_version": {
+                            "type": "string"
+                        },
+                        "meter_schema_url": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "view": {
```

This defines the stream configuration, right?

```suggestion
                "stream": {
```

---

Yes, the vocabulary you add in https://github.com/open-telemetry/opentelemetry-specification/pull/3524 helps clarify this. Will update to stream to reflect that 👍